test/unit/language-specs.js

```diff
@@ -0,0 +1,106 @@
+import sinon from 'sinon';
+import chai from 'chai';
+import XCUITestDriver from '../..';
+
+chai.should();
+
+describe('language and locale', () => {
+  const LANGUAGE = "en";
+  const LOCALE = "en_US";
+
+  describe('send only language and locale', () => {
+    it('should send translated POST /session request with valid desired caps to WDA', async () => {
+      let args = [];
```

Comment:
This can just be initialized as a literal
```js
const expectedWDACapabilities = {
  desiredCapabilities: {
    bundleId: "com.test.app",
    arguments: [
      '-AppleLanguages', `(${LANGUAGE})`,
      '-NSLanguages', `(${LANGUAGE})`,
      '-AppleLocale', LOCALE
    ],
    environment: {},
    shouldWaitForQuiescence: true,
    shouldUseTestManagerForVisibilityDetection: true,
    maxTypingFrequency: 60,
    shouldUseSingletonTestManager: true,
  }
};
```